Automattic/wp-calypso
# Pricing Grid: Add features list for each plan

#### Proposed Changes

* This PR is one among several stacked PRs that implements the new pricing grid design. In https://github.com/Automattic/wp-calypso/pull/71372, we introduced a new signup flow /start/onboarding-2023-pricing-grid that is only available in dev and wpcalypso environments.
* The design for the pricing grid can be found in 2e991-pb. Please keep the Figma design open in a separate tab.
* The parent PR is https://github.com/Automattic/wp-calypso/pull/71510.
* This PR adds the new features copy for the pricing grid and the associated styling changes as given in the Figma file. 

<img width="1634" alt="Screenshot 2022-12-26 at 1 21 05 PM" src="https://user-images.githubusercontent.com/1269602/209520808-c0c6d52f-7593-4e5f-a3d6-c830773b6def.png">

#### What Not To Test

* Responsive screens. It will be implemented in a separate PR. Right now, test in a full width desktop browser viewport.
* Non-en. The first launch will be `en` only.


#### Testing Instructions



* Navigate to `/start/onboarding-2023-pricing-grid`.
* Verify that the feature list matches Figma.
* Try with and without a custom domain, and verify the feature list UI.
* Verify that the monthly tab strikes out annual plan features.
* Verify that the production `onboarding` signup flow at `/start` is not affected by these changes.
